backend/tests/helpers/UserHelper.test.js

```diff
@@ -1,40 +1,69 @@
 const { expect } = require( 'chai' );
-const { getUserListFromSessions } = require( '../../src/helpers/UserHelper' );
+const { getUserListFromSessions } = require( '../../src/helpers/helpers' );
+
+const nowTimestamp = Date.now();
```

Comment:
You shouldn't use `Date.now()` if it's unnecessary. Fill the mocked data with plain text if possible

Reply:
Also, you should move test data to their test suites.

Reply:
Done.